browser-test/src/trusted_intermediary.test.ts

```diff
@@ -668,6 +668,66 @@ test.describe('Trusted intermediaries', () => {
     await validateScreenshot(page, 'manage-ti-group-members-page')
   })
 
+  test('sort trusted intermediaries based on selection', async ({
+    page,
+    adminTiGroups,
+  }) => {
+    await loginAsAdmin(page)
+    // add "aaa" with 3 members
+    await adminTiGroups.gotoAdminTIPage()
+    await adminTiGroups.fillInGroupBasics('aaa', 'aaa')
+    await adminTiGroups.editGroup('aaa')
+    await adminTiGroups.addGroupMember('foo@bar.com')
+    await adminTiGroups.addGroupMember('foo2@bar.com')
+    await adminTiGroups.addGroupMember('foo3@bar.com')
+    // add "bbb" with 0 members
+    await adminTiGroups.gotoAdminTIPage()
+    await adminTiGroups.fillInGroupBasics('bbb', 'bbb')
+    await adminTiGroups.expectGroupExist('bbb', 'bbb')
```

Comment:
question (non-blocking): Did you intend to check for the existence of this group in particular?

Reply:
No, this was left from a previous debugging effort.